### DISCUSSION TITLE
別害怕與眾不同

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1234] [General]
你也害怕跟大家不一樣嗎?
當大多數人選擇往右就跟著走，因為害怕成為別人眼中的異類，所以覺得要跟大家做一樣的決定。
當在討論時有別的想法卻不敢提出，因為怕大家覺得普通，所以感到沒自信。太在意別人的眼光與期待，這樣的煩惱也困擾著你嗎?
可是不一樣又怎樣? 每個人都有屬於自己的獨特，一直強迫自己和大家一樣，反而會剝奪自己的快樂。
所以保有最真實的自己吧。了解自己真正喜歡甚麼，並勇敢地表達出來，才能找到自己想走的方向。選擇和他人做一樣的事，到最後發現並不適合自己，反而會浪費時間。
當你開始認識自己，並懂得愛自己，活出自己想要的樣子，就算跟別人不同又有什麼不好的呢?

**Comment by GuanYann** [comment]
如果我們一直迎合他人的期望和要求，最終我們可能會失去自我，變得迷失和不安。相反，當我們敢於表達自己的觀點和想法時，我們會更加自信和滿足。雖然可能會遇到一些困難和挑戰，但我們也會從中學到很多東西，並最終找到屬於自己的道路和方向。

**Comment by curtis1017** [comment]
每個人都有屬於自己的獨特性，我們千萬不要有模仿的心態，看到別人哪裡比較好，就想要去學他。我覺得我們在自己的一生中，應該要去追求屬於自己的獨特性。

**Comment by Chienyuuu6** [comment]
小時候常常為因為外在的眼光而對自己的選擇感到不安，覺得什麼都要跟大家做一樣的事情，現在我覺得「獨一無二」的自己很耀眼，我有自己喜歡的事物，我喜歡跟別人與眾不同！這就是獨一無二的我，發現擁抱獨特的自己之後，自然而然生活多了很多的幸福😽

**Comment by CCello** [comment]
正是，每個人都有自己的獨特之處，這也是人格魅力的體現。當我們尊重自己的差異性，也要學會尊重他人的不同。因為只有這樣，我們才能在彼此的不同中，學習彼此的長處，互相成長，開拓更廣闊的視野和機會。
在面對困難時，我們也應該保持自信，堅定地走自己的路，因為唯有這樣，我們才能充分發揮自己的潛力，實現自己的夢想。與其因害怕被認為不一樣而選擇順從，不如以勇氣和信心，去尋找更適合自己的路。

**Comment by tower20021202** [comment]
在成長的過程中，我常常過度在乎他人的眼光，只為了努力活成社會期望的樣子，但在這過程中，我時常感到痛苦或焦慮，直到後來開始慢慢發現自己的興趣後，才發現就算跟眾人不同也沒有關係，我們本來就不可能活成每個人喜歡的樣子，但接受自己的不同後，便會發現一路上會出現許多志同到和的朋有和我們一同前行

**Comment by Jack-1D** [comment]
與眾不同是件好事。畢竟這個社會是由各式各樣的人所組成的。每個人會因為自己成長環境的不同或是站在不同的立場，而有不同的看法或見解。我認為正是因為大家的意見有衝突，才能找到最好的答案，讓社會進步。

**Comment by YoYooHuang** [comment]
我覺得人一生下來，就是不同的個體，大家就像一顆石頭一樣，可以被沖刷成不同的形狀。一切的社會價值都是人類定義出來的，但每個人的定義都大不相同，遵循自己的價值，追求自己所想要的，才是第一要點。畢竟，人一生下來，就是在走向死亡，而在這段路上，大家都應該盡全力的完成自己所認定的價值，追尋自己的夢想，活出屬於自己的一條路！

**Comment by a3240281370** [comment]
我覺得害怕與眾不同是常見情感，但我們每個人都獨一無二。能做的就是盡量接受自己、挑戰舒適區、尋找共同點，可幫助建立自信和歸屬感。給自己多點自信，將與眾不同視為優勢，改變觀念，發揮創新思維和獨特能力，我們的不同是我們的力量，勇敢展現自己的獨特之處，擁抱個人特點。

**Comment by ziihui** [comment]
害怕不同是人類的共通之處，但也應勇於展現個人獨特。跟隨主流只為避免異類眼光，卻剝奪了真正的快樂。擁抱自我，找到自己喜歡的事物，堅定地表達。選擇同樣道路可能浪費時間。了解自己、愛自己，活出真實。與眾不同又何妨？

**Comment by alicia0725** [comment]
大部分的人都還挺害怕不同的，包含我自己也是，不會太習慣當那個第一個出頭或跟大家做不同選擇的人，但我發現這樣對於我自己而言，會逐漸隨波逐流失去自己，到了大學發現越來越有機會表達自己的想法，也越能夠認清自己的獨特性。

**Comment by Alexgns** [comment]
勇敢擁抱獨特，別害怕與眾不同。在這個多元的世界裡，你的獨特價值為你帶來光彩。展現真實的自我，尋找共鳴者，共同綻放獨特之花。別讓他人的意見束縛你，堅定踏出屬於自己的路，遠離平凡，活出與眾不同的精彩人生。

**Comment by Miinteaa** [comment]
說到與眾不同，我想起小時候因為大部分的刻板印象裡，女生就是要粉紅色，害我超討厭粉色、反而喜歡藍色哈哈哈哈哈
也許這就是我天生而來反骨的部分🤣
但隨著年紀增長，現在回想起來我所做出的選擇中也有很多部分是受到他人的影響，也許我應該找回當初的勇氣，在某些事物上堅持自己的意見比較好，而不是因為多數人的決定而壓抑自己的不同

**Comment by ALAN30359704** [comment]
對我來說，我覺得要與眾不同其實是需要勇氣的，有時雖然我們說要做自己，但是當自己真的面對別人對我們的質疑或不解的時候，我覺得是我們會害怕的，因為這個社會的氛圍與規範都是人訂出來的，所以要說做自己我認為是不太可能的，因為我們只有辦法做到與少部分人一樣，而無法做到跟自己一樣。

**Comment by alstinlin** [comment]
當看事情的角度不同，得到的結果也不一樣，所以我覺得做決定不用太在意別人的眼光，只要是對自己有好處又不會影響到他人，就可以勇於承擔自己的與眾不同，相信自己的決定~

**Comment by youtinglin** [comment]
我覺得每個人都是獨一無二的個體，一定有和別人不一樣的地方，可能是優點也可能是缺點，但不管自己現在是如何的，都要先學會接納自己，並且善待自己和愛自己，使自己在成長的道路上逐漸成為想成為的樣子，世界上不可能每個人都喜歡你，但也不會每個人都討厭你，要對自己有自信，找到志同道合的朋友，也就不用擔心自己的與眾不同，而是把他們成為自己的閃光點。

**Comment by sjh501** [comment]
我覺得跟別人不一樣本來就是很正常的事情，本來在這世上就沒有相同的人，就算是同卵雙胞胎也會有所差異，更何況是其他人。雖然保有真實的自己很重要，但這個世界現在還是有一定的規則在運行，很多時候用真實的自己會有很多碰撞和痛苦，也有很多現實的束縛，這並不是每個人都能接受。而且如果都以自身為主的想法，喜歡就一起、不喜歡就不要，那就算是志同道合也會破局，對我來說這樣的結果並非是我所想要的樣子。

**Comment by Elaine2899** [comment]
我從國中就享受和別人不同，尤其當大家都覺得A是對的，這時候我就會開始思考：A真的是對的嗎？那B有沒有可能，從某個角度思考，它就會是對的呢？這樣的思考模式讓我學會從不同角度思考，不會盲目的跟從人流。矛盾的是，隨著年齡增長，我似乎變得越來越容易受到身邊朋友影響(也容易被情勒)，導致現在的我會有自己的思考但又易受身邊朋友的聲音影響，導致自己的行動常常會與想法不同，讓自己很容易心裏不舒服或不確定怎麼做才是正確的選擇。

**Comment by pudding0803** [comment]
雖然我常常都在跟別人說不要太在意其他人，其實大部分的人也沒那麼在意你（？？

但當自己遇到一些事情的時候，還是會不小心變成過於在意他人的人，當然我也會盡量說服自己，畢竟每個人的想法都不一樣，只要自己有自信就好了！

最重要的是，我不想跟大家都一樣、隨波逐流🫠

**Comment by 1104030360** [comment]
講得非常有道理，我認為一直追求自己與大眾相同是非常不好，更糟的事還會帶給我生活極大的壓力（時刻在意別人的眼光），我認為當我們專注於自己時，我覺得會放鬆許多，並且活得非常開心～

**Comment by Brownnnn78** [comment]
這裡只有我覺得當一個群體中獨特的人或是努力成為獨一無二的人很累嗎？我遇到跟自己類似的人會感覺到興奮，也很少會一直對跟自己很不一樣的人有互動，我覺得比起努力當一個獨特的人，找到自己同愛好同想法的人更重要。

**Comment by xiang305** [comment]
我自己最近也才慢慢體會到這個道理，自己在別人的眼裡並沒有我所想的重要。過去的我會很害怕因為做了甚麼事而出糗，但這樣的情況如果發生在別人身上，過了幾天，我並不會記得是誰做了這，只會記得這件事曾經發生過而已。簡單來說，我認為不必過度在意他人眼光，表現出最真實的自己就可以了
